x-pack/plugins/ingest_manager/server/services/epm/packages/assets.ts

```diff
@@ -57,7 +57,7 @@ export async function getAssetsData(
   datasetName?: string
 ): Promise<Registry.ArchiveEntry[]> {
   // TODO: Needs to be called to fill the cache but should not be required
-  const pkgkey = packageInfo.name + '-' + packageInfo.version;
+  const pkgkey = packageInfo.name + '/' + packageInfo.version;
```

Comment:
Is this required? I don't believe so, but haven't confirmed. IIRC, this is a value for a cache/map and unrelated to how things are stored anywhere else.